includes/class-wc-skroutz-analytics-tracking.php

```diff
@@ -248,4 +253,22 @@ private function calculate_tax_from_total($total, $tax_rate) {
 		return round( $total - $total / ( 1 + $tax_rate/100 ), 2 );
 	}
 
+	/**
+	 * Checks if order is prior to 30 days ago
+	 *
+	 * @return boolean
+	 *
+	 * @since    1.6.9
+	 * @access   private
+	 */
+	private function order_is_obsolete() {
+		$thirty_days_ago = (new DateTime())->modify("-30 day");
+
+		if ( is_null($this->order->get_date_created()) ) {
+			return false;
+		}
+
+		$order_creation_date = new DateTime($this->order->get_date_created()->date('Y-m-d H:i:s'));
```

Comment:
It seems like `date` method does not respect timezones.

```php
$this->order->get_date_created()->date(DateTimeInterface::ISO8601)
```

returns 2022-01-17T12:24:06+0000.

Instead we could use `format` which respects timezones.

```php
$this->order->get_date_created()->format(DateTimeInterface::ISO8601)
```

returns 2022-01-17T12:24:06+0200.

Reply:
PHP 5.4 does not accept DateTimeInterface::ISO8601. It is available in versions over 7.2. But I will keep the format method instead of date.